rllib/examples/connectors/frame_stacking.py

```diff
@@ -197,14 +197,12 @@ def _env_creator(cfg):
             grad_clip_by="global_norm",
         )
         .rl_module(
-            model_config_dict=dict(
-                {
-                    "vf_share_layers": True,
-                    "conv_filters": [[16, 4, 2], [32, 4, 2], [64, 4, 2], [128, 4, 2]],
-                    "conv_activation": "relu",
-                    "post_fcnet_hiddens": [256],
-                },
-            )
+            model_config=DefaultModelConfig(
+                vf_share_layers=True,
+                conv_filters=[(16, 4, 2), (32, 4, 2), (64, 4, 2), (128, 4, 2)],
```

Comment:
Do we need tuples or lists for the filters?

Reply:
Both are ok.